recipe/meta.yaml

```
@@ -39,25 +39,21 @@ outputs:
       build:
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
-        - armpl  # [aarch64]        
+        - armpl  # [aarch64]
```

```suggestion
        - armpl  # [aarch64]
        - meson-python
```
Should `mesonpy` be added here because it is part of the backend listed [here](https://github.com/numpy/numpy/blob/v1.26.4/pyproject.toml#L2)? Please forgive me for all my questions, I am a little rusty with package building. Lol

---

Here we only need to look at requires. The build backend mesonpy is provided by package meson-python